pkg/types/transactions_erc20.go

```diff
@@ -132,6 +132,7 @@ type (
 		ERC20InfoAPI
 	}
 
+	// ERC20InfoAPI interfaces covers all methods to retrieve the Bridge Contract's information
```

Comment:
This comment is a complete lie. it doesn't touch a contract ever. The only thing it does is:
a. get the downloader sync status
b. get the used address by the client as well as the current balance linked to that address

So you'll have to rewrite this comment to ensure it brings over the correct description.